Mdule and exports/oneModule.js

```diff
@@ -0,0 +1,14 @@
+//question-one
+var inputOne = [1, 2, 3, 4, 5];
+var callingFirst = require('./assignment1-one');
```

Comment:
@JainVayam Instead of callingFirst, callingTwo & callingThree, you can write assignmentFirst, assignmentSecond, assignmentThird. Because methods are the part of the assignment.

Note  - Variables should be specific.